README.md

```diff
@@ -141,6 +144,12 @@ This project is based on
 * [mesos-consul](https://github.com/CiscoCloud/mesos-consul)
 * [marathon-consul](https://github.com/CiscoCloud/marathon-consul)
 
+### Diferences
+
+* CiscoCloud/marthon-consul copies application information to Consul KV while allegro/marathon-consul register tasks in Consuls Services (it is more similar to CiscoCloud/mesos-consul)
+* CiscoCloud/mesos-consul uses polling while allegro/marathon-consul uses [SSE](https://mesosphere.github.io/marathon/docs/event-bus.html) to detect changes
```

Comment:
I wouldn't call it SSE as it may be related to http://www.w3schools.com/html/html5_serversentevents.asp :) Maybe just "uses Marathon's event bus" ;)